src/lib/y2network/route.rb

```diff
@@ -21,18 +21,18 @@ module Y2Network
   class Route
     # @return [IPAddr,:default] Destination; :default if it is the default route
     attr_accessor :to
-    # @return [Interface,:any] Interface to associate the route to; :any if no interface is given
+    # @return [Interface,nil] Interface to associate the route to
```

Comment:
document what nil means.

Reply:
In this case, I think it is self explanatory, it basically allows null, so not associated with any interface explicitly.